.github/workflows/realworld-bundler.yml

```diff
@@ -51,14 +51,15 @@ jobs:
           ruby-version: ${{ matrix.ruby.value }}
           bundler: none
       - name: Prepare dependencies
-        run: bin/rake spec:deps
-        working-directory: ./bundler
+        run: rake spec:deps
       - name: Replace version
-        run: BUNDLER_SPEC_SUB_VERSION=${{ matrix.bundler.value }} bin/rake override_version
+        run: BUNDLER_SPEC_SUB_VERSION=${{ matrix.bundler.value }} rake override_version
         if: matrix.bundler.value != ''
-        working-directory: ./bundler
       - name: Run Test
-        run: bin/rake spec:realworld
+        # rake spec:realworld is failing with Ruby 2.6. We should use `bin/rspec` until dropping Ruby 2.6 support.
+        run: BUNDLER_SPEC_PRE_RECORDED=1 bin/rspec --tag realworld
+        env:
+          RGV: ..
```

Comment:
I think it should pass now as it was before? Can you go back to `bin/rake spec:realworld` to verify? I'm curious about those failures.

Reply:
see https://github.com/rubygems/rubygems/actions/runs/5075333460/jobs/9116438594?pr=6691

I have no idea to investigation for next step.

Reply:
I think the reason may be that the `bin/rake` binstub has been removed, and that's what ensured that the version of RubyGems in the git repository is being used (according to the `RGV` env variable), instead of the system installed version of RubyGems. Does it work if instead of removing it, you move the binstub to the repo root `bin` folder, and use `bin/rake` instead of `rake`?

Reply:
Thanks. I try it with tomorrow.

Reply:
`bin/rake` seems working fine.